Tests/Unit/DependencyInjection/RoutePassTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\RouterBundle\Tests\Unit\DependencyInjection;
+
+use ONGR\RouterBundle\DependencyInjection\Compiler\SeoRoutesPass;
+use ONGR\RouterBundle\DependencyInjection\Compiler\SetRouterPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * Unit test for compiler pass files
+ */
+class RoutePassTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Prepares and returns the container
+     *
+     * @return ContainerBuilder
+     */
+    public function getContainer()
```

Comment:
This should be functional test. Do not mock container builder injected services. It's quite important place to test that actually all mocked functions does it's job correctly.